NOTICES

```diff
@@ -829,7 +829,7 @@ performance-now 2.1.0 : MIT License
 phadej/cabal-doctest 1.0.8 : BSD 3-clause "New" or "Revised" License
 picomatch 2.2.3 : MIT License
 picomatch 2.3.0 : MIT License
-pip 21.1.2 : MIT License
+pip 21.2.3 : MIT License
```

Comment:
@garyverhaegen-da any idea where this comes from and why it changed now? my guess is the nixpkgs update mainly due to lack of other ideas but I have no idea what it is tracking.

Reply:
merged it in https://github.com/digital-asset/daml/pull/11546, don’t really understand it but neither of the versions makes sense to me so this seems just as good.

Reply:
(apparently forgot to press "comment" yesterday)
We do get `pip`, but I'm a bit confused here. Running `bazel build //docs:docs-no-pdf` in the repo starting from a clean nix store results in ~me getting very annoyed at Nix 2.4~ `pip-21.1.3` getting installed on both c415d9d1eefe19db2488409a70dbb1e573dad05f (a random commit I happened to be on and that happens to be before the nix upgrade) and on 2efcb21f3e768f8960c832ee25324f6ba3c16143 (current main at time of testing, after nixpkgs bump). I suspect Sphinx depends on it, but I don't know why the version mismatch. Note that the version I'm seeing matches neither the existing one nor the new one.